builder/scaleway/config.go

```diff
@@ -32,6 +32,8 @@ type Config struct {
 	Bootscript   string `mapstructure:"bootscript"`
 	BootType     string `mapstructure:"boottype"`
 
+	CloudInit string `mapstructure:"cloud_init"`
```

Comment:
This should (most likely) be named `user_data` and also have `user_data_file`. See the other cloud builders for example.